### PR TITLE
Add android builder

### PR DESCRIPTION
**Pull request by johnlabarge**
This adds an Android Builder a corresponding Google Cloud Solution article is in the pipeline.

### CODE REVIEW

File: android/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Android Builder
+The Dockerfile and scripts here help you use Google Cloud Builder to build Android applications.  
+These instructions assume that you have [Android SDK](https://developer.android.com/studio/index.html) installed and can build the subject Android application locally on your workstation.  
+Steps to use:
+
+1. Create a list of the android sdk packages you need for you builds and save it to the builder directory as 'packages.txt'.  To generate the list from your workstation a script is provided:
```
Comment (ImJasonH) [inline android/README.md]: These list numbers are spaced out enough that GitHub renders them as a series of "1."s. Could you make these hand-numbered ###subheaders instead?

[the remark this answers is not in the document]
Comment (johnlabarge) [reply android/README.md]: Ugh of course.  Was working fine in my tool. 🤤

[the remark this answers is not in the document]
Comment (johnlabarge) [reply android/README.md]: This is fixed.  Changes squashed (so this is outdated)

**Comment by nof20** [comment]
Thanks so much!

**Pull request merged**